PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -5,23 +5,38 @@ public class Program
     public static void Main(string[] args)
     {
         string filePath = GetFilePath();
+
         Question[] questions = LoadQuestions(filePath);
 
+        Console.WriteLine("Welcome to Princess Bride Trivia!");
+        string PlayerName = GetPlayerName();
+
         int numberCorrect = 0;
         for (int i = 0; i < questions.Length; i++)
         {
+         
             bool result = AskQuestion(questions[i]);
             if (result)
             {
                 numberCorrect++;
+
             }
         }
+        Console.WriteLine("Congratulations " + PlayerName + "!");
         Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
     }
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        return ((float)numberCorrectAnswers / numberOfQuestions * 100) + "%";
+    }
+    public static string GetPlayerName()
+    {
+        string playerName = " ";
+        Console.WriteLine("Please enter your name: ");
```

Comment:
```suggestion
        Console.Write("Please enter your name: ");
```
nit: I think the formatting of entering on the same line as this text may be slightly cleaner